docs/src/2.4.ManagingCamera.mdx

```diff
@@ -0,0 +1,170 @@
+import CodeSandboxEmbed from './jsx/utils/CodeSandboxEmbed.js';
+
+# Managing Camera
+
+In this step, you'll learn how to:
+
+- Control camera state
+- Move the camera
+- Update the camera to follow object movements
+
+## Moving the Camera
+
+The WebGL [camera](https://webglfundamentals.org/webgl/lessons/webgl-3d-camera.html) is designed to mimic the way a human eye sees. We've developed a simple and intuitive API that abstracts away much of the math needed to control the camera via `cameraState`. Read the [Camera docs](#/docs/api/camera) to use the playground and learn more.
+
+In this step, let's add a duck model to the center of the scene, and move the camera to observe the duck from different angles.
+
+First, let's create a Duck component using [GLTFScene](#/docs/commands/gltfscene) which we'll use to compose with other command components inside Worldview later. Remember to add the actual `Duck.glb` model file in the `src/utils` folder.
+
+```js
+// src/Duck.js
+import React from "react";
+import { GLTFScene } from "regl-worldview";
+import duckModel from "./utils/Duck.glb";
+
+export default function Example() {
+  return (
+    <GLTFScene model={duckModel}>
+      {{
+        pose: {
+          position: { x: 0, y: 0, z: 0 },
+          orientation: { x: 0, y: 0, z: 0, w: 1 },
+        },
+        scale: { x: 3, y: 3, z: 3 },
+      }}
+    </GLTFScene>
+  );
+}
+```
+
+To move the camera, we will need to create a state that gets updated over time. [window.requestAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) generally matches the display refresh rate. Let's add a [React Hook](https://reactjs.org/docs/hooks-intro.html) that accepts a callback which will be called before each browser repaint.
```

Comment:
I'm not really sure what this description and the following code snippet mean. Could we add more info about what this snippet is doing, and what it will be used for, if it's not being invoked with this immediately next snippet?